### DISCUSSION TITLE
New Gemini models

### DISCUSSION THREAD

**Discussion by LotusSerene** [cline/cline#389] [Feature Requests]
gemini-1.5-pro-002 and gemini-1.5-flash-002

**Comment by saoudrizwan** [comment]
It looks like gemini-1.5-pro-latest and gemini-1.5-flash-latest aliases will use the new model versions shortly

**Comment by LotusSerene** [comment]
Alright that's good then, just have to wait it out

**Comment by saoudrizwan** [comment]
I just released v1.9.5 that points to the latest 002 models.
